src/Saritasa.Tools.SourceGenerator/Infrastructure/Options/OptionNames.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.ComponentModel;
+
+namespace Saritasa.Tools.SourceGenerator.Infrastructure.Options;
+
+/// <summary>
+/// Represents an application option names.
+/// </summary>
+internal static class OptionNames
```

Comment:
Can you please prepare a readme for this project describing how to use it, what configuration options does it have?